std/container/fixed_array.d

```diff
@@ -0,0 +1,504 @@
+module std.container.fixed_array;
+
+import core.exception, std.algorithm, std.conv, std.exception, std.range,
```

Comment:
Can most of these imports be migrated inside the struct? That way the imports won't be effected unless someone uses it.